tests/test_dse.py

```diff
@@ -2734,7 +2783,7 @@ def test_fullopt(self):
 
         assert summary1[('section0', None)].ops == 31
         assert summary1[('section1', None)].ops == 88
-        assert summary1[('section2', None)].ops == 25
+        assert summary1[('section2', None)].ops == 28
```

Comment:
what's this change due to?

Reply:
Tt goes from

```
float r4 = floorf(r7);
int posx = (int)r4;
float px = -r4 + r7;
```

To 

```
int posx = (int)(floorf(r4));
float px = r4 - floorf(r4);
```

for the interpolation section, don't think much of a difference but will double check